src/app/helpers/controller/cms-mixin.js

```diff
@@ -99,6 +100,35 @@ export async function fetchPageData({slug, preserveWrapping, setsPageTitleAndDes
     return data;
 }
 
+export function fetchPageDataJsx(fpdParams) {
```

Comment:
because this function composes hooks, it is a custom hook, and react should be yelling at you about how it doesn't start with the word `use`

Reply:
Interesting. I wonder if the lack of yelling is due to my using Preact?
I've corrected this one. Will have to be on the lookout for others.

Reply:
I've added the eslint plugin for React hooks to the next PR. That got me the yelling I was missing.